### PR TITLE
Add Apache Flink release 1.8.3

### PR DESCRIPTION
**Pull request by hequn8128**
Adds release announcement and download link for 1.8.3
Dates on announcements are still TBD and should be updated accordingly once release happens.

### CODE REVIEW

File: _posts/2019-11-27-release-1.8.3.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+date:   2019-11-27 12:00:00
+categories: news
+authors:
+- hequn:
+  name: "Hequn Cheng"
+---
+
+The Apache Flink community released the third bugfix version of the Apache Flink 1.8 series.
+
+This release includes 23 fixes and minor improvements for Flink 1.8.2. The list below includes a detailed list of all fixes and improvements.
```
[the remark this answers is not in the document]
Comment (hequn8128) [reply _posts/2019-11-27-release-1.8.3.md]: Good catch!

**Comment by hequn8128** [comment]
@dianfu Thanks, I have updated the PR.

File: _posts/2019-11-27-release-1.8.3.md [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-13726'>FLINK-13726</a>] -         Build docs with jekyll 4.0.0.pre.beta1
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-13791'>FLINK-13791</a>] -         Speed up sidenav by using group_by
+</li>
+</ul>
+        
+<h2>        Bug
+</h2>
+<ul>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-12342'>FLINK-12342</a>] -         Yarn Resource Manager Acquires Too Many Containers
```
Comment (zentol) [inline _posts/2019-11-27-release-1.8.3.md]: I went over the release notes and updated a few JIRA titles and categories; so this needs to be updated beyond FLINK-14995

Comment (hequn8128) [reply _posts/2019-11-27-release-1.8.3.md]: Thanks a lot for the update and fixing the license problem. I will update this PR once FLINK-14995 is addressed.

**Comment by hequn8128** [comment]
> Will you need someone to generate the website?

I originally planned to generate during merge, but I think we can generate the website and add an additional commit in this PR. It will be more friendly for the reviewers. What do you think?

**Comment by hequn8128** [comment]
Update the PR according to RC2. Changes mainly include:
- Change 43 fixes and minor improvements to 44.
- Update release notes
- Add an additional commit to rebuild the website. (I find the new generated content/blog/feed.xml also includes changes about 1.9.1. It seems we have missed it in the previous release, so I also add it.)

@zentol @dianfu Would be great if you can take a look. Thanks.

**Comment by hequn8128** [comment]
Update the PR according to RC3. Changes mainly include:
- Change 44 fixes and minor improvements to 45.
- Update release notes

File: _config.yml [hunk 3]
```diff
@@ -308,6 +308,10 @@ release_archive:
         version_short: 1.9
         version_long: 1.9.0
         release_date: 2019-08-22
+      -
+        version_short: 1.8
+        version_long: 1.8.3
+        release_date: 2019-11-27
```
Comment (sunjincheng121) [inline _config.yml]: Just reminder: Update the date when merging. :)

File: _posts/2019-11-27-release-1.8.3.md [hunk 4]
```diff
@@ -0,0 +1,144 @@
+---
+layout: post
+title:  "Apache Flink 1.8.3 Released"
+date:   2019-11-27 12:00:00
```
Comment (sunjincheng121) [inline _posts/2019-11-27-release-1.8.3.md]: Just reminder: Update the date when merging. :)

File: content/blog/index.html [hunk 5]
```diff
@@ -184,6 +184,21 @@ <h1>Blog</h1>
   <div class="col-sm-8">
     <!-- Blog posts -->
     
+    <article>
+      <h2 class="blog-title"><a href="/news/2019/11/27/release-1.8.3.html">Apache Flink 1.8.3 Released</a></h2>
+
+      <p>27 Nov 2019
```
Comment (sunjincheng121) [inline content/blog/index.html]: Just reminder: Update the date when merging. :)

**Comment by hequn8128** [comment]
@sunjincheng121 Thanks a lot for the review. You are right, we should update the time during merging.

**Comment by hequn8128** [comment]
@tillrohrmann  Thank you for the reminder. I will adjust the date, regenerate the docs and merge it after the release.

**Comment by wuchong** [comment]
Thanks @hequn8128 for preparing this PR. It looks good to me. 

Just a suggestion: it would be bettter to exclude the generated file (files under `content/`) from the commit. It will make the reviewing easier and avoid conflicts. 

Once 1.8.3 is released, you can rebuild website and append a commit for the generated files on this commit and merge them.

**Comment by hequn8128** [comment]
@wuchong Thanks for the review. I was thinking that adding the generated files can let reviewers check whether there is anything that has been missed. Especially this is my first time to do a release. But I think your point also makes sense as it can avoid the conflict.

**Pull request closed without merging**